lib/utilityservice.js

```diff
@@ -652,12 +652,21 @@ UtilityService.prototype.getGitChangesFromCommand = async function(gitChanges, a
 UtilityService.prototype.getGitDiffsFromOrgToLocal = async function(jobInfo) {
     try {
         var vbtDeployKey = `VBTDeployKey${jobInfo.gitCheckKey ? jobInfo.gitCheckKey : ''}`;
-        var hashKey = await this.getVlocitySetting(vbtDeployKey);
```

Comment:
can you change this to let instead of two var declarations.

Reply:
@rutlabaga: Can you check it now, if it's fine.